.meta.toml

```diff
@@ -0,0 +1,24 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
+[meta]
+template = "pure-python"
+commit-id = "3a251aab94eff9cf2aa2fd87d7266593251e4b76"
+
+[python]
+with-windows = false
+with-pypy = true
+with-future-python = false
+with-legacy-python = true
+with-docs = true
+with-sphinx-doctests = false
```

Comment:
According to the previous `tox.ini` this should be `true`.
This should also increase the test coverage.